kolibri/plugins/coach/assets/src/state/actions/exam.js

```diff
@@ -531,12 +531,7 @@ export function showExamReportPage(store, classId, examId) {
       );
     },
     error => {
-      if (error.status.code === 404) {
```

Comment:
Don't remove this logic. The idea is if a user goes to a URL like http://localhost:8000/coach/#/32d137eae73e5e4c59b4f3f90b9acdf9/exams/notarealexam123123, then the user will be redirected back to the Exams List Page.